app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -346,9 +346,10 @@ angular.module('Bastion.components').factory('Nutupane',
                 if (!column) {
                     return;
                 }
-
-                params["sort_by"] = column.id;
-                if (column.id === sort.by) {
+                if (column.id) {
+                    params["sort_by"] = column.id;
+                }
+                if (column.id === sort.by || column.id === "") {
```

Comment:
I could not think of any case where `colum id` would be `null` or `undefined`. We either have `<th bst-table-column="something">...</th>` then `column.id === "something"`. Or we have `<th bst-table-column>..</th>` then `table.column === ""`. Though I could probably change it to check falsy values to be on a safe side.